site/content/en/docs/tasks/manage/enforce_job_management/opt_in_namespace_management.md

```diff
@@ -0,0 +1,107 @@
+---
+
+This page describes how to configure Kueue to enforce opt-in namespace management, ensuring that only workloads in explicitly labeled namespaces are reconciled by Kueue.
+
+## Before you begin
+
+- Learn how to [install Kueue with a custom manager configuration](/docs/installation/#install-a-custom-configured-released-version).
+- Understand how to configure [`manageJobsWithoutQueueName`](/docs/tasks/manage/enforce_job_management/manage_jobs_without_queue_name/).
+
+## Why Use Opt-in Namespace Management?
+
+Opt-in namespace management allows cluster administrators to enforce strict quota controls by ensuring that Kueue only manages workloads in explicitly designated namespaces. This provides several key benefits:
+
+- **Prevents quota bypass**: Users cannot bypass Kueue's quota system by manually adding `queue-name` labels to workloads in unmanaged namespaces. Only workloads in opted-in namespaces are ever reconciled by Kueue, regardless of how they are labeled.
+- **Explicit control**: Cluster administrators have full control over which namespaces are subject to Kueue management by simply labeling namespaces.
+- **Consistent behavior**: This brings `batch/v1/Job` integration into alignment with `Pod`, `Deployment`, and `StatefulSet` integrations, which already enforce namespace-based filtering.
+
+## Configuration
+
+### Step 1: Label Namespaces
+
+Label the namespaces that should be managed by Kueue. For example, use the `managed-by-kueue` label:
+
+```bash
+kubectl label namespace my-namespace managed-by-kueue=true
+```
+
+### Step 2: Configure the Selector
+
+Configure the `managedJobsNamespaceSelector` in your Kueue Configuration to match the labeled namespaces. Use `matchLabels` to select namespaces that you have explicitly labeled:
+
+```yaml
+apiVersion: config.kueue.x-k8s.io/v1beta1
+kind: Configuration
+metadata:
+  name: config
+  namespace: kueue-system
+manageJobsWithoutQueueName: true
+managedJobsNamespaceSelector:
+  matchLabels:
+    managed-by-kueue: "true"
+```
+
+**Note**: When the `ManagedJobsNamespaceSelectorAlwaysRespected` feature gate is enabled, only namespaces that match the selector will be reconciled. Namespaces without the required labels will not be managed by Kueue, regardless of whether workloads in those namespaces have a `queue-name` label.
+
+### Step 3: Enable the Feature Gate
+
+Enable the `ManagedJobsNamespaceSelectorAlwaysRespected` feature gate in your Kueue manager configuration. See [Feature Gate Details](#feature-gate-details) for configuration instructions.
+
+## Feature Gate Details {#feature-gate-details}
+
+The `ManagedJobsNamespaceSelectorAlwaysRespected` feature gate controls whether the `managedJobsNamespaceSelector` restricts the reconciliation of all workloads, regardless of whether they have a `kueue.x-k8s.io/queue-name` label.
+
+### Feature Gate Status
+
+- **Alpha**: v0.13
+- **Beta**: v0.15
+
+### Enabling the Feature Gate
```

Comment:
Instead of this section please refer to the generic section https://kueue.sigs.k8s.io/docs/installation/#change-the-feature-gates-configuration

Reply:
Something like https://kueue.sigs.k8s.io/docs/concepts/cluster_queue/#lendinglimit.